variables.tf

```diff
@@ -14,12 +14,83 @@ variable "aws_region" {
   type        = string
 }
 
+variable "aws_access_key_id" {
+  description = "AWS access key id"
+  type        = string
+}
+
+variable "aws_secret_access_key" {
+  description = "AWS secret access key"
+  type        = string
+}
+
+variable "aws_role_arn" {
+  description = "AWS role arn"
+  type        = string
+}
+
+variable "aws_s3_enabled" {
```

Comment:
reuse feature_blob_storage_enabled

Reply:
Similarly for all the other variables. Corresponding flags already exist cc @dunefro